Backend-Challenges.md

```diff
@@ -48,6 +48,21 @@ Generally speaking, an NFT consists of following components:
 
 The goal of this challenge is to flesh out the constructor, have it accept the arguments passed in the test, and make it set the max supply inside the constructor to what was passed as an argument.
 
+### Solidity libraries and openzeppelin
+
+In Solidity, you can import libraries that will make life easier. OpenZeppelin is a company that provides free standardized contract libraries to use, such as ERC721. These contracts are firmly audited, meaning they are generally secure to implement in your contracts.
+
+ERC721 is a standard for NFT smart contracts. There are others, such as ERC1155 by Enjin and ERC721A by Azuki, each with its own set of functionalities. For the workshop, ERC721 NFTs will suffice.
+
+- https://docs.openzeppelin.com/contracts/3.x/
+- https://docs.openzeppelin.com/contracts/3.x/api/token/erc721
+
```

Comment:
I'd recommend linking to the EIP as well, so they can always refer to the official standard. 
Openzeppelin is good, but not perfect. See: https://nftchance.medium.com/the-gas-efficient-way-of-building-and-launching-an-erc721-nft-project-for-2022-b3b1dac5f2e1

Reply:
Done